src/__tests__/downshift.misc-with-utils-mocked.js

```diff
@@ -0,0 +1,52 @@
+// this is stuff that I couldn't think fit anywhere else
+// but we still want to have tested.
+
+import React from 'react'
+import {mount} from 'enzyme'
+import Downshift from '../'
+import {scrollIntoView} from '../utils'
+
+jest.useFakeTimers()
+jest.mock('../utils')
+
+test('does not scroll from an onMouseEnter event', () => {
+  class HighlightedIndexController extends React.Component {
+    state = {highlightedIndex: 10}
+    handleStateChange = changes => {
+      if (changes.hasOwnProperty('highlightedIndex')) {
+        this.setState({highlightedIndex: changes.highlightedIndex})
+      }
+    }
+    render() {
+      return (
+        <Downshift
+          onStateChange={this.handleStateChange}
+          highlightedIndex={this.state.highlightedIndex}
+          render={({getInputProps, getItemProps}) => (
+            <div>
+              <input {...getInputProps()} />
+              <div {...getItemProps({item: 'hi', 'data-test': 'item-1'})} />
+              <div {...getItemProps({item: 'hey', 'data-test': 'item-2'})} />
+            </div>
+          )}
+        />
+      )
+    }
+  }
+  const wrapper = mount(<HighlightedIndexController />)
+  const input = wrapper.find('input')
+  const item = wrapper.find(sel('item-1'))
+  item.simulate('mouseenter')
+  jest.runAllTimers()
```

Comment:
I learned a lot from this test. I didn't know jest had these functions. That's pretty cool!